lib/cfme/cloud_services/data_collector.rb

```diff
@@ -41,22 +39,55 @@ def raw_manifest
 
     {
       "cfme_version" => cfme_version,
-      "manifest"     => {
-        "openstack" => common.clone,
-        "rhevm"     => common.clone,
-        "vmwarews"  => common.clone,
+      "manifest" => {
+        "core" => {
+          "Zone" => {
+            "name" => nil
+          }
```

Comment:
This is an example of collecting non-provider oriented data (e.g. Zone).

Reply:
We might want to add in the cfme guid here if we're going to create a cfme source to hang this off of in topology?

Reply:
Same for the providers actually

Reply:
Interestingly, I don't think we have a product-wide guid... I checked in MiqDatabase and MiqEnterprise.  We have MiqRegion.guid, but that's per-region and we have MiqServer.guid, but that's per-server.

@bdunne @jrafanie Thoughts? Am I missing something or should we add a guid column to MiqDatabase or MiqEnterprise?

Reply:
The only product-wide values we have are the first startup generated tokens for session/csrf protection so you can reset it in one place.

What would this one be used for?  Would it also be generated at "first" startup?